src/app/globals.css

```diff
@@ -142,13 +127,6 @@
   background-color: red;
 }
 
-/* link */
-.utrecht-link {
```

Comment:
Vraagje: waarom deze styling weghalen bij `.utrecht-link` en verplaatsen naar `.utrecht-page-footer__content`? Het is styling die voor alle links van toepassing is toch? 

Ik zag in de preview dat alle links wel de styling krijgen zoals verwacht, dus volgens mij is het geen probleem voor de uitkomst van het project. Maar als iemand link styling wil toevoegen/aanpassen is het misschien onduidelijk dat ze dan bij de `utrecht-page-footer__content` class moet zijn?🤔

Reply:
Alleen de links in de footer hebben de inverse color scheme.